### PR TITLE
Add support for Ukrainian language.

### PR DESCRIPTION
**Pull request by mik9**
#### Database Migration
NO

#### Description
Add ability to download series in Ukrainian language.

#### Todos
- [x] Tests
- [x] Documentation

#### Issues Fixed or Closed by this PR

*

### CODE REVIEW

File: src/NzbDrone.Core.Test/ParserTests/LanguageParserFixture.cs [hunk 1]
```diff
@@ -225,6 +225,13 @@ public void should_parse_language_czech(string postTitle)
             result.Language.Id.Should().Be(Language.Czech.Id);
         }
 
+        [TestCase("Castle.2009.S01E14.Ukrainian.HDTV.XviD-LOL")]
+        [TestCase("True.Detective.S01E01.1080p.WEB-DL.Urk.Eng.TVKlondike")]
```
Comment (Thirrian) [inline src/NzbDrone.Core.Test/ParserTests/LanguageParserFixture.cs]: The testcase says `urk` and the regex matches on `ukr`...

Comment (mik9) [reply src/NzbDrone.Core.Test/ParserTests/LanguageParserFixture.cs]: Oops. Have not been able to run test locally...

**Comment by mik9** [comment]
Btw, I did similar changes as for [Radarr](https://github.com/Radarr/Radarr/pull/3594/files) but I still don't see Ukrainian language in the Language Profile.

Is there other places I missed for the new UI?

File: src/NzbDrone.Core/Languages/Language.cs [hunk 2]
```diff
@@ -81,6 +81,7 @@ public override bool Equals(object obj)
         public static Language Hebrew       { get { return new Language(23, "Hebrew"); } }
         public static Language Lithuanian   { get { return new Language(24, "Lithuanian"); } }
         public static Language Czech        { get { return new Language(25, "Czech"); } }
+        public static Language Ukrainian    { get { return new Language(26, "Ukrainian"); } }
 
         
         public static List<Language> All
```
Comment (markus101) [inline src/NzbDrone.Core/Languages/Language.cs]: Ukranian needs to be added here to show up in new profiles.

**Pull request closed without merging**